## Supplementary scripts and datasets

### *Selection, Periodicity and Potential Function for Highly Iterative Palindrome-1 (HIP1) in Cyanobacterial Genomes*

by Minli Xu, Jeffrey G. Lawrence, and Dannie Durand

Accepted by **Nucleic Acids Research**   (Last updated: 1-11-2018)

---

### Code and scripts
1. Second order Markov chain estimate of the expected number of motifs within a genome (perl) [[link]](https://github.com/minli-xu/HIP1/tree/master/expected_number)
2. Motif conservation between a pair of aligned genomes (perl) [[link]](https://github.com/minli-xu/HIP1/tree/master/compute_conservation)
3. MatLab scripts for periodicity analysis and sine wave curve fitting [[link]](https://github.com/minli-xu/HIP1/tree/master/periodicity)

### Datasets
1. Genomic coordinates for HIP1 and control hexamers  (with randomized coordinates) [[link]](https://github.com/minli-xu/HIP1/tree/master/HIP1_coordinates_and_random)
2. Parameters estimated from sine wave fitting for HIP1 and control hexamers (with randomized coordinates) [[link]](https://github.com/minli-xu/HIP1/tree/master/HIP1_periodicity_parameters)
3. Transcripts dataset from [Vijayan et. al. (2011) paper](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-5-r47) (with transcripts ACEu scores) [[link]](https://github.com/minli-xu/HIP1/tree/master/oshea_dataset)

### Other software used
1. DNA Master [[Link]](http://cobamide2.bio.pitt.edu/computer.htm)
2. MAUVE [[Link]](http://darlinglab.org/mauve/mauve.html)

---

For questions, please contact [Dr. Dannie Durand](https://www.cmu.edu/bio/people/faculty/durand.html).

[Durand Lab](http://www.cs.cmu.edu/~durand/) - [Department of Biological Sciences](https://www.cmu.edu/bio/) - [Carnegie Mellon University](https://www.cmu.edu/)
